rules/pkg-normalize.js

```diff
@@ -22,6 +22,10 @@ const isGitRepository = repository => {
 		return false;
 	}
 
+	if (typeof repository !== 'string' && !Array.isArray(repository) && Object.prototype.hasOwnProperty.call(repository, 'url')) {
```

Comment:
I thought about this for a while. `repository` has to be an object or a `string`. If it's neither one of those, it means it is not a (git) repository and the function should return false.

```js
if (typeof repository !== 'string' && typeof repository !== 'object') {
    return false;
}

if (typeof repository === 'object' && repository.url) {
    repository = repository.url;
}

return repository.includes('github.com') || /^\w+\/\w+$/.test(repository);
```